### DISCUSSION TITLE
harbor 2.2.4 prometheus alert rules

### DISCUSSION THREAD

**Discussion by finalha** [goharbor/harbor#16871] [Q&A]
I'm using the harbor 2.2.4 and config the harbor exporter, now prometheus can get the metrics of harbor. 
Some metrics can be found here: https://goharbor.io/docs/2.2.0/administration/metrics/
But I can't find the offical wiki of prometheus alert rules of harbor. Where could I get it?

**Comment by QiuToo** [comment]
我看Harbor官方也没有警告规则，如果您熟悉Prometheus，你可以尝试自己编写这些规则，欢迎探讨这个问题。
